fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -153,7 +167,53 @@ public WebTarget addQueryParameter(WebTarget target, String header, String val)
     public Response doGet(String path, String mimeType) {
         WebTarget target = getWebTarget();
         target = target.path(path);
-        return target.request(mimeType).get(Response.class);
+        return target.request(mimeType)
+                .header("X-FHIR-TENANT-ID", tenantName)
+                .header("X-FHIR-DSID", dataStoreId)
+                .get(Response.class);
+    }
+    
+    private void verifyDownloadUrl(String downloadUrl) {
+        // Minio doesn't support file level ACL using which we make can make the published file public, so we have to 
+        // get the token first, and then use the token to download the file. 
```

Comment:
this would be a good limitation for us to document if we ever advertise our support for Minio

Reply:
updated user guide.